DOCSP-31521: V2 cleanup (#85)
mongodb/docs-rust: source/fundamentals/aggregation.txt

API Documentation
~~~~~~~~~~~~~~~~~

> To learn more about any of the methods or types mentioned in this

Reviewer: ```suggestion
To learn more about the methods and types mentioned in this
```
